templates/admin/forms/list.html.twig

```diff
@@ -0,0 +1,157 @@
+        </div>
+    </div>
+    {% if error %}
+        <div class="alert alert-danger">
+            {{ error|translate }}
+        </div>
+    {% endif %}
+    <div class="row">
+        <div class="col-xs-12">
+
+            <!-- Nav tabs -->
+            <ul class="nav nav-tabs" role="tablist">
+                <li role="presentation" class="active">
+                    <a href="#registered"
+                       aria-controls="home" role="tab"
+                       data-toggle="tab">
+                        {{ "Registered"|translate }}
+                        &nbsp;<span class="badge">{{ registered|length }}</span>
+                    </a>
+                </li>
+                <li role="presentation">
+                    <a href="#unregistered"
+                       aria-controls="profile" role="tab"
+                       data-toggle="tab">
+                        {{ "Unregistered"|translate }}
+                        {# Wrap the badge in an if statement so bootstrap will
+                           hide element if the count is empty
+                         #}
+                        &nbsp;<span class="badge">{% if unregistered|length > 0 %}{{ unregistered|length }}{% endif %}</span>
+                    </a>
+                </li>
+            </ul>
+
+            <div class="tab-content">
+                <div role="tabpanel" class="tab-pane active" id="registered">
+                    <table class="table table-striped table-condensed">
+                        <thead>
+                        <tr>
+                            <th>&nbsp;</th>
+                            <th>Name</th>
+                            <th>Status</th>
+                            <th colspan="2">&nbsp;</th>
+                            <th>{{ "Priority"|translate }}</th>
+                            <th>{{ "Category"|translate }}</th>
+                            <th>{{ "Nickname"|translate }}</th>
+                            <th>{{ "Access Control"|translate }}</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        {% for row in registered %}
+                            <tr>
+                                <td class="text-muted"><small>{{ row.id }}</small></td>
+                                <td><strong>{{ row.name }}</strong></td>
+                                <td class="{{ row.statusColor }}">
+                                    <i class="{{ row.statusIcon }}"></i>&nbsp;
+                                    {% if row.status == "Registered" %}
+                                        {{ row.status|translate }}
+                                    {% else %}
+                                        <a class="text-primary" href="./forms_admin.php?id={{ row.id|escape('url') }}&method={{ row.statusMethod|escape('url') }}">{{ row.status|translate }}</a>
```

Comment:
something to clarify;
since auto escape is on, is this getting bot escaped for url and html (note I don't know the answer twofold; meaning I don't know if it is doing both and I don't know if we should be doing both; research needed :)).